content/project/board-election-process.adoc

```diff
@@ -16,7 +16,7 @@ More contributors might be added to the election committee if approved by the li
 The charter of the Jenkins board is to ensure the health of not only the software project, but of the communities of plugin developers and users.
 To do this effectively, members of the board must bring various perspectives to the table: what are the needs of users; of committers; of organizations, both large and small; of commercial interests.
 As the composition of the Jenkins board changes over time, we want to ensure that this balance of viewpoints is maintained.
-Current board members best understand this balance, and so are in the best position to select candidates for inclusion onto the board.
+Current board members best understand this balance, and are in the best position to select candidates for inclusion onto the board.
```

Comment:
```suggestion
Current board members best understand this balance, and are in the best position to select candidates for inclusion into the board.
```
(not sure about this one)

Reply:
```suggestion
Current board members understand this balance, and are in the best position to select candidates for inclusion on the board.
```
I changed it to on to try and remove some of the ambiguity of into/onto, and removed the first usage of best as it seems redundant with the next half of the sentence.
Does that work for what you had in mind?

Reply:
I'm still a bit confuse about the inclusion "on" the board instead of what I would instinctively used, "in" the board, but that sound nicer as it is.

(I thought the same about "best" but didn't find a way to fix it.)

Reply:
No worries @lemeurherve I'm going to do a bit of digging and see if I can find a solid answer on that. I can get some assistance from my fellow writers to ensure it's correct.

Reply:
I updated that sentence with the help of @b-fisher-cb, and decided to re-word it for clarity

Reply:
You mean you've made the problem disappears ^^
I like this final formulation, good job!

Reply:
Thanks @lemeurherve, all thanks to teamwork!